traffic/__init__.py

```diff
@@ -15,6 +15,11 @@
 __version__ = importlib_metadata.version("traffic")  # type: ignore
 __all__ = ["config_dir", "config_file", "cache_dir"]
 
+logger_traffic = logging.getLogger("traffic")
+logger_traffic.setLevel(logging.WARNING)
+
+logger: logging.Logger = logging.getLogger(__name__)
```

Comment:
why do you need two loggers here ?
More specifically can't we just have one logger named traffic rather than one logger for each file ?

Reply:
The idea was to have a local one and set up the global one. I just realized that `logger: logging.Logger = logging.getLogger(__name__)` in the `__init__.py` file does not return a logger named `traffic.__init__` but directly `traffic` resp. `traffic.core` in the `core/__init__.py` file.

Fixed that.

> More specifically can't we just have one logger named traffic rather than one logger for each file ?

The problem then is to find out from where the log message was raised. Technically that would also be possible but I think it makes things confusing. Furthermore when having a specific logger for each file one can (but not have to) disable and enable the logging of certain parts individually.

Reply:
No problem :) Thanks for this library which saved me a lot of time :)